### PR TITLE
Clarifai-LiteLLM integration

### PR DESCRIPTION
**Pull request by mogith-pn**
1. Added clarifai as LLM provider in LiteLLM.
2. This enables to use standard prompt for all models and removes the dependancy to pass additional tokens like "INST", "SYSTEM" "HUMAN"in the prmopts.
3. Allows to retrieve response in standard format of OpenAI API response.

### CODE REVIEW

File: cookbook/liteLLM_clarifai_Demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#install necessary packages\n",
+    "!pip install litellm\n",
+    "!pip install clarifai"
```
Comment (sanjaychelliah) [inline cookbook/liteLLM_clarifai_Demo.ipynb:26]: From the implementation in `clarifai.py`, I see that you are directly calling the `Post model outputs ` API and so there is no need for importing clarifai SDK I think.

You have provided the `api_base` and `api_key` params in the main `completion` function. So setting the base api to default prod api and getting the **PAT** from either _os.environ_ or through the _api_key_ param in `completion` function will be ideal.

Comment (mogith-pn) [reply cookbook/liteLLM_clarifai_Demo.ipynb:26]: Yeah but neither for other providers , so this is just to drive the SDK downloads. 
all providers mentioned their package and not used inside so we can also mention it in examples.

Comment (sanjaychelliah) [reply cookbook/liteLLM_clarifai_Demo.ipynb:26]: Mm. Not sure, we are adding it as a prerequisite even though its not. Yeah if thats the strategy everyone's following, then fine. 👍

**Review by sanjaychelliah — APPROVED** [review]
👍

**Pull request merged**